src/lwc/geTemplateBuilderFormField/geTemplateBuilderFormField.html

```diff
@@ -49,6 +50,7 @@
                                          variant="label-hidden"
                                          data-qa-locator={qaLocatorCheckboxRequired}>
                         </lightning-input>
+                        <span class="slds-assistive-text" id="requiredCheckboxDescription">{labelGeAssistiveRequiredCheckboxDescription}</span>
```

Comment:
Deploying this now.. Just to confirm, hardcoding the `id` like this isn't a problem because ids and aria attributes in the same template are linked automatically right? Prior to the `id` value being transformed by the platform that is. Is that correct?

Reply:
Yep it gets transformed to be local to the individual instance of the component we're in.